### PR TITLE
Update testing.rst

### PR DESCRIPTION
**Pull request by jeffblack360**
I was thinking a blurb about making PHPUnit available globally might be convenient for readers?

### CODE REVIEW

File: en/development/testing.rst [hunk 1]
```diff
@@ -37,6 +37,19 @@ tests::
...
+    As a convenience you can make phpunit.phar available globally
+    on Unix or Linux with the following:
+    
+    chmod +x phpunit.phar
+    
+    sudo mv phpunit.phar /usr/local/bin/phpunit
+    
+    phpunit --version
+    
+    Please refer to the PHPUnit documenation for instructions regarding
```
Comment (dereuromark) [inline en/development/testing.rst]: typo documentation

**Comment by ravage84** [comment]
While this is conventient if you have only one application/project or all of your applications depend on the same version of PHPUnit, this could become problematic/confusing if you try to test an older application that needs PHPUnit 3.x (which CakePHP 2.x does).

So I'm not sure if we should recommend this oldskool style setup...

**Comment by jeffblack360** [comment]
I hadn't considered that ravage84. Thanks!

**Comment by markstory** [comment]
I think its reasonable to give people options. PHPUnit as a system binary, doesn't seem any worse than composer as a system binary.

**Pull request merged**